tests/tasks/setup_mock_wifi_wpa3_sae.yml

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Install packages required to set up mock wifi network
+  package:
+    name:
+      - NetworkManager
+      - wpa_supplicant
+    state: present
+
+# Even though hostapd can be installed via EPEL 8, Simultaneous Authentication
+# of Equals (SAE) has not been enabled by default. To warrant the test support
+# on RHEL(dist-tag), we setup hostapd copr repo to enable SAE option.
```

Comment:
What do you mean with `RHEL(dist-tag)` - the `(dist-tag)` does not seem to make sense to me.

Reply:
I mean `EPEL`  and the package with `EPEL` tag can be installed on `RHEL` and `CentOS` via copr.

Reply:
Changed into 
```
# Even though hostapd can be installed via EPEL 8, Simultaneous Authentication
# of Equals (SAE) has not been enabled by default. To warrant the test support
# on CentOS 8, we setup hostapd copr repo to enable SAE option.
``